<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SAS</string>
  </property>
  <widget class="QLabel" name="LSAS1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SAS IP Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TSAS1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="LsTemp1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LTemp1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Template Name</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="RTemp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Template</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Rred1">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Redirection</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TFile1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LFile1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>File Name</string>
   </property>
  </widget>
  <widget class="QComboBox" name="LsTemp2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LSAS2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SAS IP Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="LTemp2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Template Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TSAS2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="Rred2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Redirection</string>
   </property>
  </widget>
  <widget class="QLabel" name="LFile2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>File Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TFile2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="RTemp2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Template</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="value">
    <number>40</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Next_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chAdd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Another Server</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
